src/website/apps/blog/views.py

```
@@ -0,0 +1,55 @@
+                description='Ordering fields such as order, published_at, or title',
+            ),
+        ],
+    ),
+    retrieve=extend_schema(
+        summary='Retrieve a blog post',
+        description='Returns a single published blog post using the public identifier or legacy numeric ID.',
+    ),
+)
+class BlogPostViewSet(SlugModelViewSetMixin, viewsets.ReadOnlyModelViewSet):
+    queryset = BlogPost.objects.all()
+    permission_classes = [IsAuthenticatedOrReadOnly]
```

Ruff (0.15.12)

[warning] Mutable default value for class attribute

(RUF012)